lib/Command/Room/Create.php

```diff
@@ -156,6 +156,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			$output->writeln(sprintf('<error>%s</error>', $e->getMessage()));
```

Comment:
Not an error :) use info or question?

Reply:
I don't did this. I can change but is an error, the methods addRoom* trigger errors and this row is to show the error.

Reply:
Nevermind wrong line